src/Elcodi/Component/User/Entity/Interfaces/LastLoginInterface.php

```diff
@@ -36,5 +36,5 @@ public function getLastLoginAt();
      *
      * @return $this Self object
      */
-    public function setLastLoginAt($lastLoginAt);
+    public function setLastLoginAt(\DateTime $lastLoginAt);
```

Comment:
We should avoid complete namespaces here. Instead of that, we should use here `DateTime` here, adding as well the use statemenet on the top of the file (Everywhere but Traits, remember)